src/test/java/redis/clients/jedis/tests/commands/StreamsCommandsTest.java

```diff
@@ -233,7 +233,9 @@ public void xgroup() {
     
     jedis.xgroupDestroy("xgroup-stream", "consumer-group-name");
 
-    //TODO test xgroupDelConsumer
+    //xgroupDelConsumer
```

Comment:
Not required.

Reply:
> Even though this breaks, but it should be considered as a bugfix.

it is.

Reply:
What is your review the standards of your code?I doubt it.

Reply:
> I can remove it, but what is your review the standards of your code?I doubt it.

@dengliming I'm afraid I didn't get you.

However, putting a comment `//method` before calling `method()` is absolutely nonsensical.

Reply:
Sorry,It's my misunderstanding.but the project is moving too slowly,many of the changes were suspended.